keps/sig-api-machinery/2855-automated-stored-versions/README.md

```diff
@@ -87,24 +87,25 @@ It is desirable for the [kube-storage-version-migrator] to modify a CRD's `statu
 
 ### Goals
 
-- Allow the [kube-storage-version-migrator] to modify a CRD's `status.storedVersions` after a successful migration.
+- Allow the [kube-storage-version-migrator] to safely modify a CRD's `status.storedVersions` after a successful migration.
+- Stop other actors from modifying the storage version while a migration is ongoing.
 
 ## Proposal
 
-In order for the [kube-storage-version-migrator] to safely modify a CRD's `status.storedVersions` field after a successful migration, this KEP proposes that the [kube-storage-version-migrator] introduce a [Validating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) that prevents changes to a CRD's `spec.versions[*].storage` fields during a migration.
+In order for the [kube-storage-version-migrator] to safely modify a CRD's `status.storedVersions` field after a successful migration, this KEP proposes that the [kube-storage-version-migrator] introduces a [Validating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) that prevents changes to a CRD's `spec.versions[*].storage` fields during a migration.
```

Comment:
failure policy is important to specify here

Reply:
It is visible in the manifest, but would make it explicit here as it drives the failiure mode discussion.